tests/plugins/webp-uploads/picture-element-tests.php

```diff
@@ -0,0 +1,91 @@
+<?php
+/**
+ * Tests for webp-uploads plugin picture-element.php.
+ *
+ *
+ * @group picture-element
+ *
+ * @package webp-uploads
+ */
+
+use PerformanceLab\Tests\TestCase\ImagesTestCase;
+
+class WebP_Uploads_Picture_Element_Tests extends ImagesTestCase {
+	/**
+	 * Test that images are wrapped in picture element when enabled.
+	 *
+	 * @dataProvider data_provider_it_should_maybe_wrap_images_in_picture_element
+	 *
+	 * @test
+	 *
+	 * @param bool $jpeg_and_webp          Whether to enable JPEG and WebP output.
+	 * @param bool $picture_element        Whether to enable picture element output.
+	 * @param bool $expect_picture_element Whether to expect the image to be wrapped in a picture element.
+	 */
+	public function it_should_maybe_wrap_images_in_picture_element( bool $jpeg_and_webp, bool $picture_element, bool $expect_picture_element ): void {
```

Comment:
Eliminating `@test`:

```suggestion
	 *
	 * @param bool $jpeg_and_webp          Whether to enable JPEG and WebP output.
	 * @param bool $picture_element        Whether to enable picture element output.
	 * @param bool $expect_picture_element Whether to expect the image to be wrapped in a picture element.
	 */
	public function test_maybe_wrap_images_in_picture_element( bool $jpeg_and_webp, bool $picture_element, bool $expect_picture_element ): void {
```